app/pathogen/sarscov2/dashboard/filters.tsx

```diff
@@ -1,112 +1,83 @@
-import { useQuery, useQueryClient } from "@tanstack/react-query";
-import {
-  SarsCov2ActionType,
-  SarsCov2Context,
-  SarsCov2ContextType,
-} from "@/contexts/sarscov2-context/sarscov2-context";
-import useSarsCov2Data from "@/hooks/useSarsCov2Data";
+import { Filters } from "@/components/customs/filters";
+import { FilterableField } from "@/components/customs/filters/available-filters";
+import { useNewSarsCov2Data } from "@/hooks/useNewSarsCov2Data";
+import { SarsCov2Context } from "@/contexts/pathogen-context/pathogen-contexts/sc2-context";
+import { useSarsCov2Filters } from "@/hooks/useSarsCov2Filters";
 
-const addFilterMulti = (
-  value: string[],
-  newFilter: string,
-  state: SarsCov2ContextType,
-  data: any
-) => {
-  state.dispatch({
-    type: SarsCov2ActionType.UPDATE_FILTER,
-    payload: {
-      filter: newFilter,
-      value: value,
-      // Consider making this a static list synced with the backend so that context is static and we do not need to do this. 
-      data: data ? data : [],
-    },
-  });
-};
+interface SarsCov2FiltersProps {
+  className?: string;
+}
 
-const getHeader = (filterKey: string) => {
-  const filterHeader = filterKey.replaceAll("_", " ");
-  return filterHeader.charAt(0).toUpperCase() + filterHeader.slice(1);
-};
+export const SarsCov2Filters = (props: SarsCov2FiltersProps) => {
+  const state = useContext(SarsCov2Context);
+  const { data } = useNewSarsCov2Data();
+  const { data: filterData } = useSarsCov2Filters();
 
-const buildFilterDropdown = (
-  filter: string,
-  placeholder: string,
-  state: SarsCov2ContextType,
-  filterOptions: string[],
-  data: any
-) => {
+  const dateFilters = [
+    FilterableField.samplingStartDate,
+    FilterableField.samplingEndDate
+  ]
 
-  return (
-      <div className="pb-3" key={filter}>
-        <Select
-        key={filter}
-          handleOnChange={(value) =>
-            addFilterMulti(value, filter, state, data)
-          }
-          heading={placeholder}
-          selected={state.selectedFilters[filter] ?? []}
-          options={filterOptions.filter((assay: string) => assay != null)}
-          optionToLabelMap={{}}
-        />
-      </div>
-  );
-};
+  const studyLocationFilters = [
+    FilterableField.whoRegion,
+    FilterableField.unRegion,
+    FilterableField.country,
+  ];
 
-export function Filters() {
-  const state = useContext(SarsCov2Context);
+  const demographicFilters = [
+    FilterableField.ageGroup,
+  ];
 
-  const { data } = useSarsCov2Data();
+  const testInformationFilters = [
+    FilterableField.scope,
+    FilterableField.sourceType,
+    FilterableField.antibodies,
+    FilterableField.testType,
+    FilterableField.isotypes
+  ];
 
-  const filters = useQuery({
-    queryKey: ["SarsCov2Filters"],
-    queryFn: () =>
-      fetch(`${process.env.NEXT_PUBLIC_API_BASE_URL}/sarscov2/filter_options`).then(
-        (response) => response.json()
-      ),
-  });
+  const filterSections = [{
```

Comment:
Haven't confirmed with anyone, just taking my best guess. We'll iterate on this, let's just get something down to start.